cmd/transcriber/call/tracks.go

```diff
@@ -138,6 +140,24 @@ func (t *Transcriber) processLiveTrack(track trackRemote, sessionID string, user
 			continue
 		}
 
+		// We ignore out of order packets as they would cause synchronization
+		// issues. In the future we may want to reorder them but that requires us to keep
+		// buffers and complicate the whole process.
+		if pkt.Timestamp < prevRTPTimestamp {
+			slog.Debug("out of order packet",
+				slog.Int("diff", int(pkt.Timestamp)-int(prevRTPTimestamp)),
+				slog.String("trackID", ctx.trackID))
+
+			// Check that timestamp hasn't wrapped around. Fairly unlikely but it's
+			// a possibility since the starting timestamp is generated randomly so
+			// it could be close to the end of the uint32 range.
```

Comment:
Randomly?  https://datatracker.ietf.org/doc/html/rfc7587#section-4.1 : `The RTP
   timestamp corresponds to the sample time of the first encoded sample
   in the encoded frame.`
The word "corresponds" could mean a lot of things, so maybe it is random... 
Is there a ref for that?

Reply:
Of course, from the RTP spec, https://datatracker.ietf.org/doc/html/rfc3550#section-5.1:

> The initial value of the timestamp SHOULD be random, as for the
      sequence number. 

Also the `pion` implementation is a good reference: https://github.com/pion/rtp/blob/7dc2af56736b663e76f1400ba403532ba590bceb/packetizer.go#L47